components/opponent/commons/opponent_display.lua

```diff
@@ -228,9 +230,15 @@ end
 function OpponentDisplay.BlockPlayers(props)
 	local opponent = props.opponent
 
-	local playerNodes = Array.map(opponent.players, function(player)
-		return PlayerDisplay.BlockPlayer(Table.merge(props, {player = player, team = player.team}))
-			:addClass(props.playerClass)
+	--only apply note to first player, hence extract it here
```

Comment:
Why would the note only be applied to the first player? Might be a bit misleading if it is actually a note about the whole party

Reply:
because imo it looks shit otherwise

Reply:
Expected something like this, tbh i don't have a better idea on display